File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/kotlin/ADV-01_C.kt
fun main() {
    println("ADV-START")
    System.out.flush()
    val a: Long = 9223372036854775807L
    val v: Int = a.toInt()
    println("OBS=V:$v")
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/cpp/ADV-01_C.cpp
#include <cstdio>
#include <cstdint>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    std::int64_t x = 9223372036854775807;
    std::int32_t v = x;
    printf("OBS=V:%d\n", (int)v); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}
